src/CollAction/Views/Projects/Find.cshtml

```diff
@@ -9,118 +9,7 @@
     var defaultDisplayModel = FindProjectViewModel.DefaultDisplayProjectViewModel;
 }
 
-<h2>@Localizer["Find"]</h2>
```

Comment:
I looked at this page for a react proof of concept. Since all of it's HTML is due to be replaced anyway I left it out.